web/dashboard/src/views/settings/settings.jsx

```diff
@@ -290,17 +292,18 @@ function Users({ users, scmUrl, deleteUser }) {
                 </div>
               </div>
             }
-            <div className="flex items-center">
-              <svg xmlns="http://www.w3.org/2000/svg"
-                onClick={() => {
-                  // eslint-disable-next-line no-restricted-globals
-                  confirm(`Are you sure you want to delete ${user.login}?`) &&
-                    deleteUser(user.login);
-                }}
-                className="items-center cursor-pointer inline text-red-400 hover:text-red-600 opacity-70 h-4 w-4" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 7l-.867 12.142A2 2 0 0116.138 21H7.862a2 2 0 01-1.995-1.858L5 7m5 4v6m4-6v6m1-10V4a1 1 0 00-1-1h-4a1 1 0 00-1 1v3M4 7h16" />
-              </svg>
-            </div>
+            {user.login !== login &&
+              <div className="flex items-center">
+                <svg xmlns="http://www.w3.org/2000/svg"
+                  onClick={() => {
+                    // eslint-disable-next-line no-restricted-globals
+                    confirm(`Are you sure you want to delete ${user.login}?`) &&
+                      deleteUser(user.login);
+                  }}
+                  className="items-center cursor-pointer inline text-red-400 hover:text-red-600 opacity-70 h-4 w-4" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 7l-.867 12.142A2 2 0 0116.138 21H7.862a2 2 0 01-1.995-1.858L5 7m5 4v6m4-6v6m1-10V4a1 1 0 00-1-1h-4a1 1 0 00-1 1v3M4 7h16" />
+                </svg>
+              </div>}
```

Comment:
Minor thing: please move `}` to the next line, and also fix conflicts so I can merge.